docs/sources/http_api/folder_dashboard_search.md

```diff
@@ -97,3 +97,17 @@ Content-Type: application/json
   }
 ]
 ```
+## cURL examples
+
+The most basic example for a dashboard for which there is no authentication required is:
```

Comment:
This is not a general case: thee reason that there is no authentication required in the above example is because in the play site is enabled [the anonymous access](https://grafana.com/docs/grafana/latest/auth/#anonymous-authentication).

Also I'm not sure it's generally good to include an example of a request which does not require authentication towards our play site.

Reply:
Should I just remove the whole example? Can you suggest a better one?

Reply:
I would say yes. The easier way to perform API requests is using the credentials with basic authentication which is already included.
The alternative would be to suggest to enable anonymous access and try:
`curl http://localhost:3000/api/search` but it's no more a basic example.

Whatever you think is better.

Reply:
Updated. Please make sure what I wrote is correct.